packages/ckeditor5-dev-build-tools/src/config.ts

```diff
@@ -0,0 +1,254 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { existsSync, readFileSync } from 'fs';
+import { createRequire } from 'module';
+import type { PackageJson } from 'type-fest';
+import { type Plugin, type RollupOptions } from 'rollup';
+import { getPath } from './utils.js';
+import type { BuildOptions } from './build.js';
+
+/**
+ * Rollup plugins
+ */
+import json from '@rollup/plugin-json';
+import styles from 'rollup-plugin-styles';
+import terser from '@rollup/plugin-terser';
+import svg from 'rollup-plugin-svg-import';
+import commonjs from '@rollup/plugin-commonjs';
+import typescriptPlugin from '@rollup/plugin-typescript';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { banner as bannerPlugin } from './plugins/banner.js';
+import { replace as replacePlugin } from './plugins/replace.js';
+import { translations as translationsPlugin } from './plugins/translations.js';
+
+/**
+ * PostCSS plugins
+ */
+import postcssImport from 'postcss-import';
+import postcssMixins from 'postcss-mixins';
+import postcssNesting from 'postcss-nesting';
+
+/**
+ * If build speed becomes an issue, we can replace typescript with swc. However, this
+ * would put the responsibility of checking the types and generating `.d.ts` files on
+ * the package author.
+ *
+ * In the below configuration, swc is also used for minification and mangling, but it
+ * doesn't do as well as terser. Both options should be checked before deciding to use
+ * one or the other.
+ *
+ * swc( {
```

Comment:
` ```js ` is missing (perhaps).

Reply:
This comment is internal and will not be used for documentation, so there is no need for that :wink: